ReactiveCocoa/Swift/Property.swift

```diff
@@ -97,6 +109,16 @@ public final class MutableProperty<Value>: MutablePropertyType {
 		}
 	}
 
+	/// A signal that will send the property's changes over time,
+	/// then complete when the property has deinitialized.
+	public lazy var signal: Signal<Value, NoError> = { [unowned self] in
```

Comment:
This is fine. Eventually I think we should actually expose this from `SignalProducer`.